index.html

```diff
@@ -0,0 +1,477 @@
+              <a href="#">Notifications</a>
+            </li>
+            <li class="list-item">
+              <div class="icon-wrapper">
+                <i class="fa-regular fa-envelope"></i>
+              </div>
+              <a href="#"> Messages</a>
+            </li>
+            <li class="list-item">
+              <div class="icon-wrapper">
+                <i class="fa-regular fa-bookmark"></i>
+              </div>
+              <a href="#">Bookmarks</a>
+            </li>
+            <li class="list-item">
+              <div class="icon-wrapper">
+                <i class="fa-regular fa-rectangle-list"></i>
+              </div>
+              <a href="#">Lists</a>
+            </li>
+            <li class="list-item">
+              <div class="icon-wrapper"><i class="fa-regular fa-user"></i></div>
+              <a href="#">Profile</a>
+            </li>
+            <li class="list-item">
+              <div class="png"></div>
+              <a href="#">More</a>
+            </li>
+            <div class="tweet-btn">
+              <img class="feather" src="/icons/feather.png" alt="feather" />
+              <button div="tweet-btn" id="tweet-btn">Tweet</button>
+            </div>
+          </ul>
+          <div class="profile three-column-grid__expand-two">
+            <div class="profile-image">
+              <img src="/images/profile.png" alt="profile-image" />
+            </div>
+            <div class="profile-name">
+              <h4 class="text-md-bold">Tibor Toth</h4>
+              <p class="text-md">@Tibor4343234334</p>
+            </div>
+            <div class="more">
+              <i class="fa-solid fa-ellipsis"></i>
+            </div>
+          </div>
+        </aside>
+        <div class="center-wrapper">
+          <main class="center">
+            <div class="header-wrapper">
+              <div class="header-icon profile-image">
+                <img
+                  class="image-header"
+                  src="/images/profile.png"
+                  alt="profile image"
+                />
+                <h4 class="text-heading-bold">Home</h4>
+              </div>
+
+              <div class="icon">
+                <img src="/icons/top-tweets.png" alt="tweets" />
+              </div>
+            </div>
+            <div
+              class="center-header border-bottom three-column-grid__expand-two"
+            >
+              <div class="profile-image header-image border-bottom">
+                <img src="/images/profile.png" alt="profile-images" />
+              </div>
+              <div class="header-post border-bottom">
+                <div class="header-question text-heading cl-fade">
+                  <textarea
+                    type="text"
+                    id="textarea"
+                    name="w3review"
+                    rows="2"
+                    cols="50"
+                    placeholder="What's happening?"
+                  ></textarea>
```

Comment:
what does textarea do?

Reply:
It creates an area where you can type, like when you write your tweet.